cmd/etcd-launcher/main.go

```diff
@@ -338,14 +349,13 @@ func (e *etcdCluster) removeDeadMembers() error {
 		if member.Name == e.config.podName {
 			continue
 		}
-		healthy, err := e.isEndpointHealthy(member.PeerURLs[0])
-		if err != nil {
-			continue
-		}
-		if !healthy {
-			if _, err := e.client.MemberRemove(context.Background(), member.ID); err != nil {
-				return err
-			}
+
+		if err = wait.Poll(1*time.Second, 30*time.Second, func() (bool, error) {
+			return e.isEndpointHealthy(member.PeerURLs[0])
+		}); err != nil {
+			log.Infof("member [%s] is not responding, removing from cluster", member.Name)
```

Comment:
Please use:
`log.Infow("member is not responding, removing from cluster", "member-name", member.Name)`